stl/src/winapisupp.cpp

```diff
@@ -305,7 +305,7 @@ extern "C" _Success_(return > 0 && return < BufferLength) DWORD
 
 #else // ^^^ defined(_ONECORE) / !defined(_ONECORE) vvv
 
-static int __cdecl initialize_pointers() {
+static int __cdecl initialize_pointers() noexcept {

```

Comment:
This does not appear to be `extern "C"`. I think it's a good change but it's a drive-by and deserves mention in the PR description (I am increasingly uncomfortable with mixing these drive-by `noexcept` additions into this large PR, whereas the drive-by comment addition for `_Thrd_abort` is fine).